packages/jwt-verifier/lib.js

```diff
@@ -63,9 +63,10 @@ class OktaJwtVerifier {
   constructor(options = {}) {
     // Assert configuration
     assertIssuer(options.issuer, options.testing);
-    assertClientId(options.clientId);
+    if( options.clientId ) { 
```

Comment:
Do we even need `assertClientId` anymore? Or should we just let this be handled by the custom claims assertion logic?

Reply:
The value here is catching "{clientId}" cases.